addons/point_of_sale/static/src/xml/Screens/ClientListScreen/ClientListScreen.xml

```diff
@@ -45,6 +45,7 @@
                                                 <th t-if="!env.isMobile">Address</th>
                                                 <th t-if="!env.isMobile" class="client-line-email">Contact</th>
                                                 <th t-if="env.isMobile">ZIP</th>
+                                                <th class="pos-right-align" t-if="isBalanceDisplayed">Balance</th>
```

Comment:
Do you really need this commit? Can't we just use some xpath in the pos_loyalty module?

Reply:
hello @pimodoo thank you for your suggestion here. The reason that we followed this approach is because the "Balance" column displays two fields that are inherited from two different modules: `loyalty_points` from pos_loyalty and `total_due` from pos_settle_due. So, in my mind, it seemed simpler to have a "base" column in point_of_sale to attach the inherited fields. 

The other approach would be that each module that is installed (they may be installed one separately from the other) would have to look for a Balance column (with an xpath expression) and if the column didn't exist (already created from the other module) it would have to create it itself. This, has the benefit that the code related to each module is self-contained, meaning that all code that implements pos_loyalty or pos_settle_due features exists inside them, and there is no extra code in point_of_sale to support their functionality (as we have done in the current approach). However, it might be a little bit more complex to implement. 

I am very open to following each of these approaches. What is your opinion? Thank you for considering this! :)

Reply:
@elkr-odoo ok, no problem, you can  keep it

Reply:
Ok, great! thank you!